spec/tobias/payout_spec.rb

```diff
@@ -1,16 +1,18 @@
 require "rails_helper"
+require_relative "factories/payout_factory"
+require_relative "factories/beneficiary_factory"
 
 RSpec.describe Tobias::Payout do
   describe "#issue" do
     it "issues a Payment to each Beneficiary for their share of the #payout_amount" do
       payout = create(:tobias_payout, payout_amount_cents: 150_00)
 
-      beneficiaries = create_list(10, :tobias_beneficiary, trust: payout.trust)
```

Comment:
Apparently my "wishful thinking" was a bit  too wishful. I inverted the argument order fore `create_list` here. I wish my editor could tell me which arguments FactoryBot methods took; but I haven't taken the time to figure out how to make it do that.